GitUI/CommitInfo/CommitInfo.Designer.cs

```diff
@@ -171,23 +177,69 @@ private void InitializeComponent()
+            this.tlpnlRevisionHeader.Size = new System.Drawing.Size(782, 54);
+            this.tlpnlRevisionHeader.TabIndex = 4;
+            // 
             // _RevisionHeader
             // 
             this._RevisionHeader.BackColor = System.Drawing.SystemColors.ControlLight;
             this._RevisionHeader.BorderStyle = System.Windows.Forms.BorderStyle.None;
             this._RevisionHeader.ContextMenuStrip = this.commitInfoContextMenuStrip;
             this._RevisionHeader.Dock = System.Windows.Forms.DockStyle.Fill;
-            this._RevisionHeader.Location = new System.Drawing.Point(109, 2);
+            this._RevisionHeader.Location = new System.Drawing.Point(4, 2);
             this._RevisionHeader.Margin = new System.Windows.Forms.Padding(4, 2, 4, 2);
             this._RevisionHeader.Name = "_RevisionHeader";
             this._RevisionHeader.ReadOnly = true;
-            this._RevisionHeader.Size = new System.Drawing.Size(780, 93);
+            this._RevisionHeader.Size = new System.Drawing.Size(666, 50);
             this._RevisionHeader.TabIndex = 0;
             this._RevisionHeader.Text = "";
             this._RevisionHeader.ContentsResized += new System.Windows.Forms.ContentsResizedEventHandler(this._RevisionHeader_ContentsResized);
             this._RevisionHeader.LinkClicked += new System.Windows.Forms.LinkClickedEventHandler(this.RevisionInfoLinkClicked);
             this._RevisionHeader.MouseDown += new System.Windows.Forms.MouseEventHandler(this._RevisionHeader_MouseDown);
             // 
+            // tagSignPicture
+            // 
+            this.tagSignPicture.Image = global::GitUI.Properties.Resources.commit_ok;
+            this.tagSignPicture.Location = new System.Drawing.Point(677, 3);
+            this.tagSignPicture.Name = "tagSignPicture";
+            this.tagSignPicture.Size = new System.Drawing.Size(48, 48);
+            this.tagSignPicture.SizeMode = System.Windows.Forms.PictureBoxSizeMode.AutoSize;
+            this.tagSignPicture.TabIndex = 2;
+            this.tagSignPicture.TabStop = false;
+            this.tagSignPicture.Visible = false;
+            this.tagSignPicture.Click += new System.EventHandler(this.tagSignPicture_Click);
+            // 
+            // commitSignPicture
+            // 
+            this.commitSignPicture.Image = global::GitUI.Properties.Resources.commit_ok;
+            this.commitSignPicture.Location = new System.Drawing.Point(731, 3);
+            this.commitSignPicture.Name = "commitSignPicture";
+            this.commitSignPicture.Size = new System.Drawing.Size(48, 48);
+            this.commitSignPicture.SizeMode = System.Windows.Forms.PictureBoxSizeMode.AutoSize;
+            this.commitSignPicture.TabIndex = 3;
+            this.commitSignPicture.TabStop = false;
+            this.commitSignPicture.Visible = false;
+            this.commitSignPicture.Click += new System.EventHandler(this.commitSignPicture_Click);
```

Comment:
both images need to have mouse cursor changed to hand to indicate it is actionable element
and they need tooltips too

Reply:
added event handler to change mouse icon.